### ISSUE TITLE
htmlnorm.c OOB write

### ISSUE THREAD

**Issue by felixgr** [vrtadmin/clamav-devel#34]
``` 621 →       →       *dst = '\0';``` here a out-of-bounds write 1 occurs with static byte NULL.

Repro (base64 -d and bunzip2 compressed):

```
QlpoOTFBWSZTWUkizMQAAAV/2cAgQAAJAkHKZLBQwUAPr+FwCWABARAQAAgAQAGsAHESaGjIaaAD
QAaAAMZDIaDQaNAGgAaGCapFKekAAANAPU0yNMgaUBoAAAAAAGlhxjzlOugOMRgh6ZYiHIgOgT4l
WhGUFVtzquTBP4QyS8micKiZip6MVV7pGYVGKnLgK1gykySaSZBV+daxEwpLKr5YX9wHZg4lJZxg
q10gq5nDerNGYFhXRkq2pN5SlrKX3wkm8Uqbgqw+uIt7ia0HTkWpVyyq1wKSxUV48hVuJiTAVYkw
psppJmgVaBVy2FXXlBLeWYCszKJhTCkssEykV0YGkmFMwjMRYSl1ZOIKv2CrXXBV/Obl1dvZBV29
2Crv/6CAbOfCIxgREQA/F3JFOFCQSSLMxA==
```

**Comment by steve-morgan** [comment]
Tracking with https://bugzilla.clamav.net/show_bug.cgi?id=11441.

**Comment by felixgr** [comment]
Was there any progress? Can you add groebert at google.com to the ACL of the bugzilla bug?

**Comment by steve-morgan** [comment]
The fix for this issue shipped with ClamAV 0.99.1.
